src/dmd/expression.d

```diff
@@ -5094,6 +5097,30 @@ extern (C++) final class VectorExp : UnaExp
     }
 }
 
+extern (C++) final class VectorArrayExp : UnaExp
```

Comment:
A comment describing what this is would be useful, as there's no way to tell from the following code.